sdk/servicebus/azure-messaging-servicebus/src/test/resources/session-records/getQueue.json

```diff
@@ -12,7 +12,7 @@
+      "Body" : "<entry xmlns=\"http://www.w3.org/2005/Atom\"><id>https://sb-java-conniey-1.servicebus.windows.net/queue-5?enrich=true&amp;api-version=2021-05</id><title type=\"text\">queue-5</title><published>2022-11-09T03:17:33Z</published><updated>2022-11-15T08:48:26Z</updated><author><name>sb-zejia2</name></author><link rel=\"self\" href=\"https://sb-zejia2.servicebus.windows.net/q-zejia?enrich=true&amp;api-version=2021-05\"/><content type=\"application/xml\"><QueueDescription xmlns=\"http://schemas.microsoft.com/netservices/2010/10/servicebus/connect\" xmlns:i=\"http://www.w3.org/2001/XMLSchema-instance\"><LockDuration>PT30S</LockDuration><MaxSizeInMegabytes>1024</MaxSizeInMegabytes><RequiresDuplicateDetection>false</RequiresDuplicateDetection><RequiresSession>false</RequiresSession><DefaultMessageTimeToLive>P14D</DefaultMessageTimeToLive><DeadLetteringOnMessageExpiration>false</DeadLetteringOnMessageExpiration><DuplicateDetectionHistoryTimeWindow>PT10M</DuplicateDetectionHistoryTimeWindow><MaxDeliveryCount>20</MaxDeliveryCount><EnableBatchedOperations>true</EnableBatchedOperations><SizeInBytes>11985584</SizeInBytes><MessageCount>59942</MessageCount><IsAnonymousAccessible>false</IsAnonymousAccessible><AuthorizationRules></AuthorizationRules><Status>Active</Status><CreatedAt>2022-11-09T03:17:33.977Z</CreatedAt><UpdatedAt>2022-11-15T08:48:26.613Z</UpdatedAt><AccessedAt>2022-11-10T07:09:16.985Z</AccessedAt><SupportOrdering>true</SupportOrdering><CountDetails xmlns:d2p1=\"http://schemas.microsoft.com/netservices/2011/06/servicebus\"><d2p1:ActiveMessageCount>59942</d2p1:ActiveMessageCount><d2p1:DeadLetterMessageCount>0</d2p1:DeadLetterMessageCount><d2p1:ScheduledMessageCount>0</d2p1:ScheduledMessageCount><d2p1:TransferMessageCount>0</d2p1:TransferMessageCount><d2p1:TransferDeadLetterMessageCount>0</d2p1:TransferDeadLetterMessageCount></CountDetails><AutoDeleteOnIdle>P10675199DT2H48M5.477S</AutoDeleteOnIdle><EnablePartitioning>false</EnablePartitioning><EntityAvailabilityStatus>Available</EntityAvailabilityStatus><EnableExpress>false</EnableExpress><MaxMessageSizeInKilobytes>1024</MaxMessageSizeInKilobytes></QueueDescription></content></entry>",
```

Comment:
There might be changes to other session recordings as well, please make sure these test are functional

Reply:
all tests are passed in CI and my local environment. Are there other things that I need to do?